WalletWasabi.Documentation/BackendDeployment.md

```diff
@@ -54,22 +57,6 @@ adduser user
 usermod -aG sudo user
 ```
 
-### Increase the number of files limit
```

Comment:
Do not remove this part. It is critically important.

Reply:
It was moved to the service file below. Editing files in `/etc` that you didn't create yourself sometimes leads to annoying questions during upgrades.